gson/src/main/java/com/google/gson/GsonBuilder.java

```diff
@@ -349,6 +349,10 @@ public GsonBuilder setFieldNamingPolicy(FieldNamingPolicy namingConvention) {
    * Configures Gson to apply a specific naming strategy to an object's fields during
    * serialization and deserialization.
    *
+   * <p>The Gson instance might only use the field naming strategy once for a field and
```

Comment:
I think I would use `{@link Gson}` here to make it clear that we are talking about the class here (in the previous paragraph we are talking about Gson the project).

Reply:
I had a similar concern, being afraid that users might think Gson statically (for all `Gson` instances) caches the results. That is why I chose "Gson instance", but I have now changed it to "The created Gson instance". In the context of this builder I hope this should be clear enough.

In this specific line where you added the comment I don't think a `{@link Gson}` would be needed. The previous paragraph is also about the `Gson` instance and not about specifying a naming strategy globally. Or did I misunderstand you?

Reply:
This is fine now. I would probably use `{@link}` or `{@code}` every time we're talking about `Gson` the class, since Gson is also the name of the project. But that's probably overly fussy.